tools/inference_engine_tests/test_cases/test_model.cpp

```diff
@@ -178,7 +182,7 @@ TEST(ModelTest, MatMul_fused_activation)
     // ask model for output size (we know that there has to be 1 output in this test case)
     inference_engine_tensor_mapping_t output_mapping{};
     ASSERT_EQ(inferenceEngineModelGetOutputs(model, &output_mapping, nullptr), true);
-    ASSERT_EQ(output_mapping.id, port_out);
+    ASSERT_EQ(output_mapping.id, port_matmul_out); // we fuse to matmul + activations...
```

Comment:
User doesn't know about any fusions details, so from user perspective port_out should be valid all the time.
There are 2 solutions to this problem:
a) Simple: we never fuse (optimize out) output nodes
b) More complex: we need to preserve internal node_id and overwrite it for the GPUNode.

I prefer to keep it simple and use option #a

Reply:
ok, make sense, in final graph structures we'd probably have some "out_port" nodes, not just straight operators as we have right now... so keeping it simple should be a better option.

Reply:
one question: how should we check in tests (so from the client code) if nodes were fused? if we want to do it, then we have to expose some getCompileInfo objects that would contains some useful information. The fusion/optimialization could be completely transparent to users, but then we'd have no easy way of testing it...